tests/e2e/replication/show_while_creating_invalid_state.py

```diff
@@ -697,7 +697,7 @@ def test_sync_replication_when_main_is_killed():
         )
 
         # 2/
-        QUERY_TO_CHECK = "MATCH (n) RETURN COLLECT(n.name);"
+        QUERY_TO_CHECK = "MATCH (n) RETURN COUNT(n.name);"
```

Comment:
why did you put count here?

Reply:
Because this test relies that aggregation will return some result. This PR basically does the change that collect(null) is not any more [] but rather nothing while count(null) is still 0